.github/workflows/common/common_pytest.yml

```diff
@@ -0,0 +1,39 @@
+# A common CI workflow that runs pytest on all actively supported Python versions.
+#
+# Assumptions:
+#
+# 1. The repo works with the below-specified version of Poetry.
+# 2. The repo can be tested with pytest. 
+# 3. All Python versions named below are enabled for testing.
+
+name: Python Pytest Check (MeltanoLabs)
+
+on: [push]
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9, 3.10]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+      with:
+        version: 1.3.2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'poetry'
+    - name: Install dependencies
+      run: |
+        poetry install
+    - name: Test with pytest
+      run: |
+        poetry run pytest --capture=no
```

Comment:
Why run with `-s`/`--capture=no` by default? That's generally only recommended for local debugging - it can mess up the test output, and even break some tests.

Reply:
I think it's so the output from the tap's `discover`, etc. isn't displayed since it can contain a large number of lines

Reply:
Wouldn't setting `-s` cause the output to be displayed? The default is for Pytest to capture and handle all output sent to stdout/stderr.

Reply:
Ah you're right, so maybe that's AJ's intention here. I do prefer to suppress that output.

Reply:
No, this is just legacy. The "--capture=no" translates basically to "print=all" as you say above, and I agree that's probably not what we want here. (For security reasons as well as log length reasons.)

Initially, this was probably written to print output for debugging purposes, but that was more of a short-term aid than a long-term solution. I'll remove.

Reply:
Done. ✅